### PR TITLE
wappalyzer: Add support for CSS inspection/patterns

### PR DESCRIPTION
**Pull request by kingthorin**
Part of zaproxy/zaproxy#6180

- Added necessary infrastructure/code.
- Added unit tests.
- Add CHANGELOG notes.
- Rolled version identifier to 21.0.0 (since now targeting ZAP 2.10,
which is somewhat breaking).

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Review by psiinon — APPROVED** [review]

**Comment by thc202** [comment]
Does it need to target 2.10? (Fine either just asking in case worth releasing one last time to 2.9.)

File: addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/Application.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
     private List<AppPattern> url = new ArrayList<AppPattern>();
     private List<AppPattern> html = new ArrayList<AppPattern>();
     private List<Map<String, AppPattern>> metas;
+    private List<AppPattern> csses = new ArrayList<AppPattern>();
```
Comment (thc202) [inline addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/Application.java:39]: Could be singular? Would match more of the others.

File: addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerPassiveScanner.java [hunk 2]
```diff
@@ -181,6 +194,7 @@ private void checkUrlMatches(HttpMessage msg) {
     private void addIfMatches(AppPattern appPattern, String content) {
         List<String> results = appPattern.findInString(content);
         if (results != null) {
+            System.out.println(content);
```
Comment (thc202) [inline addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerPassiveScanner.java:197]: Remove.

**Comment by kingthorin** [comment]
Done.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thanks!

**Comment by kingthorin** [comment]
Sorry I missed your earlier question, I targeted 2.10 to get the new isCss methods.

**Pull request merged**